tools/streamer_recorder/include/recorder.cpp

```diff
@@ -0,0 +1,173 @@
+#include "recorder.h"
```

Comment:
Why is this cpp file in include directory?

Reply:
Moving out up one dir all cpp files from include...

Reply:
DONE